OneCommunityGlobal/HighestGoodNetworkApp
# Changhao - Unit tests for userSearchPanel

# Description
added new unit tests for userSearchPanel.jsx

## How to test:
1. check into current branch
2. do `npm install` and `npm test UserSearchPanel.test.js` to run this PR locally
3. verify all 12 tests passed

## Screenshot:
<img width="677" alt="Screenshot 2023-12-08 at 8 24 45 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/104778270/c60a8a9c-ffc3-4a04-aa52-f391657ce934">
